All_home_works/hw8/task2.py

```diff
@@ -0,0 +1,64 @@
+"""
+Description in task2.rst
+"""
+import sqlite3
+
+
+class Data:
+    def __init__(self, cursor):
+        self.cursor = cursor
+        self.column_names = [x[0] for x in self.cursor.description]
+
+    def __next__(self):
+        try:
+            if data := self.cursor.fetchone():
+                self.set_attributes_from_row(self.column_names, data)
+                return self
+            raise StopIteration
+        except sqlite3.ProgrammingError:
+            raise StopIteration
+
+    def set_attributes_from_row(self, column_names, row):
+        for key, value in zip(column_names, row):
+            setattr(self, key, value)
+
+    def __getitem__(self, item):
+        return self.__dict__[item]
+
+    def __eq__(self, other):
+        return other in self.__dict__.values()
+
+
+class TableData:
+    def __init__(self, db_name, db_table):
+        self.conn = sqlite3.connect(db_name)
+        self.cursor = self.conn.cursor()
+        self.db_table = db_table
+
+    def __iter__(self):
+        self.cursor.execute(f"SELECT * FROM '{self.db_table}'")  # noqa: S608
+        return Data(cursor=self.cursor)  # noqa: S608
```

Comment:
What is S608?

Reply:
S608 Possible SQL injection vector through string-based query construction

Reply:
Can we fix it in this line?

Reply:
`        return Data(cursor=self.cursor)  # noqa: S608`
in this line is not actually needed. Are you talking about the previous line?

Reply:
>  in this line is not actually needed. Are you talking about the previous line?

No, I have left comments precisely to this line.  It's a bad practice to add things that are not actually needed.